devsetup/tripleo/ceph.sh

```diff
@@ -62,3 +62,12 @@ openstack overcloud ceph deploy \
     --network-data network_data.yaml \
     --cephadm-default-container \
     --output deployed_ceph.yaml
+
+# TODO: jgilaber workaround to have ceph working with only 1 compute node
+# should remove it once we move to have 3 computes
+ssh -o StrictHostKeyChecking=no root@192.168.122.103 cephadm shell -- ceph osd pool create images --size 1
```

Comment:
-1 

Please remove lines 65-73 and then update line 60 so it passes `--single-host-defaults`. It should look like this:

openstack overcloud ceph deploy \
  --single-host-defaults \
  --tld localdomain \

This will configure ceph ceph as described below since you're using a single-node cluster:

  https://docs.ceph.com/en/latest/cephadm/install/#single-host

As per the above `osd_pool_default_size = 2` will be set and every pool created (when `openstack overcloud deploy` runs will inherit that pool size).

You can tweak the initial ceph configuration further by passing an initial ceph conf as described here:

https://docs.openstack.org/project-deploy-guide/tripleo-docs/latest/features/deployed_ceph.html#ceph-configuration-options

In general, tripleo be passed the desired ceph configuration so that you don't need to SSH in and run these commands.

Reply:
thanks @fultonj  for the link, that is much cleaner. I'll try first with `--single-host-defaults` and if that's not enough I'll add a ceph.conf